web-console/src/views/query-view/query-view.tsx

```diff
@@ -196,6 +196,10 @@ export class QueryView extends React.PureComponent<QueryViewProps, QueryViewStat
       return Api.instance.post(`/druid/v2${isSql ? '/sql' : ''}`, payload, { cancelToken });
     });
 
+    const generateQueryId = (): string => {
```

Comment:
I think druid query ids are normally UUID v4 by convention. it would be best to stick to that. Use an off the shelf uuid v4 library like https://www.npmjs.com/package/uuid instead on hand rolling a random number generator here.

Reply:
Noted it ! install uuid --save and use it to generate our queryId